pkg/operator/sync_test.go

```diff
@@ -118,3 +123,242 @@ func TestWaitForDeploymentRollout(t *testing.T) {
 		})
 	}
 }
+
+func TestSyncValidatingWebhooks(t *testing.T) {
+	defaultWebhook := mapiv1.NewValidatingWebhookConfiguration()
```

Comment:
is pretty confusing to me that we use `default**Webhook**` to name a `WebhookConfiguration` and then immediately after we deepcopy we use a different suffix and call it `expected**Configuration**` , and then in the testCases  `exisingWebhook`
can we please use always the same suffix if we are referring to the same struct type? or just don't use suffix at all e.g default, current/existing/got, expected...

Reply:
Let me drop the suffix.